src/api/app/controllers/webui/staging_workflows/staging_projects_controller.rb

```diff
@@ -0,0 +1,55 @@
+module Webui
+  module StagingWorkflows
+    class StagingProjectsController < WebuiController
+      layout 'webui2/webui'
+
+      before_action :require_login
+      before_action :set_staging_workflow
+      after_action :verify_authorized
+
+      def create
+        authorize @staging_workflow
+
+        staging_project_name = "#{@staging_workflow.project}:Staging:#{params[:staging_project_name]}"
+        staging_project = @staging_workflow.staging_projects.build(name: staging_project_name)
+
+        if staging_project.valid? && staging_project.store
+          flash[:success] = "Staging project with name = \"#{staging_project_name}\" was successfully created"
```

Comment:
Minor thing, but I would find it more logical / clearer to use staging_project.name since that's what really is created. Even though it's unlikely that the resulting project name is something different.

Reply:
Done!